### ISSUE TITLE
[Bug]: Fix install components 

### ISSUE THREAD

**Issue by abushahmaa** [radix-vue/shadcn-vue#60]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0
Shadcn Vue version: 1.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

https://www.shadcn-vue.com/docs/components/input.html

### Steps to reproduce

when i try to install https://www.shadcn-vue.com/docs/components/input.html

### Describe the bug

when i am try to install button, input any other any other components it got this error

"npx shadcn-vue@latest add input
Need to install the following packages:
  shadcn-vue@0.1.5
Ok to proceed? (y) y
node:internal/modules/cjs/loader:1048
  const err = new Error(message);
              ^

Error: Cannot find module 'detype'
Require stack:
- C:\Users\91983\AppData\Local\npm-cache\_npx\360157ef5dc9b5fb\node_modules\shadcn-vue\dist\index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at file:///C:/Users/91983/AppData/Local/npm-cache/_npx/360157ef5dc9b5fb/node_modules/shadcn-vue/dist/index.js:121:21
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:308:24)
    at async loadESM (node:internal/process/esm_loader:42:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\91983\\AppData\\Local\\npm-cache\\_npx\\360157ef5dc9b5fb\\node_modules\\shadcn-vue\\dist\\index.js'
  ]
}

Node.js v20.6.1"

### Expected behavior

how to solve this issue ?

### Conext & Screenshots (if applicable)

![error](https://github.com/radix-vue/shadcn-vue/assets/72514035/aecf38e9-3628-49df-be99-697969c74890)

**Comment by madsh93** [comment]
yeah i'm getting the same one when `npx shadcn-vue@latest init`

**Comment by m-glass** [comment]
yep same with `yarn dlx` and `npx` on mac

**Comment by zernonia** [comment]
@Dunqing  since v0.1.4 (merging https://github.com/radix-vue/shadcn-vue/pull/45) causes the above issue 🙈

**Comment by Dunqing** [comment]
I will take a look at this

**Comment by MahediAmin** [comment]
<img width="695" alt="image" src="https://github.com/radix-vue/shadcn-vue/assets/26061598/70e4d8f0-b613-4e3e-a21e-31ac15bf16ee">
same problem

**Comment by Dunqing** [comment]
### Workaround

Typescript users should use version 0.1.4 for the time being.

```shell
// init
npx shadcn-vue@0.1.4 init

// add
npx shadcn-vue@0.1.4 add
```

**Comment by zernonia** [comment]
> ### Workaround
> Typescript users should use version 0.1.4 for the time being.
> 
> ```shell
> // init
> npx shadcn-vue@0.1.4 init
> 
> // add
> npx shadcn-vue@0.1.4 add
> ```

@Dunqing  this is not working since 0.1.4

**Comment by MahediAmin** [comment]
in my nuxt project i init project by choose js not typescript .
but when i  npx shadcn-vue@0.1.4 add button or any other's component it will come's ts format 
how i get only js format

**Comment by zernonia** [comment]
Thanks @Dunqing . We pushed a fix for this in `0.1.6`.

However I realize the we currently dont have full support for JS.
The reason is that we are currently defining props/emit type using TS. It would not work for JS unfortuantely.

This would required some major changes in all of the components. https://github.com/radix-vue/shadcn-vue/issues/62
